aii-opennebula/src/main/perl/opennebula.pm

```diff
@@ -23,6 +23,7 @@ use constant DOMAINNAME => "/system/network/domainname";
 use constant MAXITER => 20;
 use constant TIMEOUT => 30;
 use constant MINIMAL_ONE_VERSION => version->new("4.8.0");
+use constant ONE_DEFAULT_URL => 'http://localhost:2633/RPC2';
```

Comment:
@stdweird should we switch to `Readonly` the rest of the constant values as well? `MINIMAL_ONE_VERSION`, `TIMEOUT`...